ui/src/components/core/input/input-file.tsx

```diff
@@ -0,0 +1,79 @@
+import { Flex } from '../../layout';
+import { dripStitches } from '../../../theme/stitches';
+import { useRef, useState } from 'react';
+import { Icon } from '../icon';
+import { StyledErrorMessage } from './input-error-message';
+
+const { styled } = dripStitches;
+
+const BorderInput = styled('div', {
+  borderStyle: 'solid',
+  borderColor: '$gray7',
+  width: '$22',
+  height: '$22',
+  transition: 'border-color 0.2s ease-in-out',
+  borderWidth: '$default',
+  borderRadius: '$lg',
+  zIndex: '$docked',
+  my: '$1h',
+
+  '&:hover': {
+    borderColor: '$gray8',
+  },
+});
+
+const validateFileSize = (file: File) => {
+  const fileSize = file.size / 1024; // in KB

```

Comment:
Here the `1024` could be stored as a config constant. Also would be interesting to have at least a comment explaining why we have the size validation to this size.